extra/utils/validation.js

```diff
@@ -0,0 +1,88 @@
+let user=[
+    {
+        traineeEmail: 'trainee1@successive.tech',
+        reviewerEmail: 'reviewer1@successive.tech'
+    },
+    {
+        traineeEmail: 'aryan.singhal@successive.tech',
+        reviewerEmail: 'rahul.sadhukhan@successive.tech'
+    },
+    {
+        traineeEmail: 'trainee1@gmail.com',
+        reviewerEmail: 'reviewer1@hotmail.com'
+    },
+    {
+        traineeEmail: 'Aman$2@successive.tech',
+        reviewerEmail: 'vinay.chaudhary@successive.tech'
+    }
+];
+let regex=/[a-z]([[-]*\w+[.]*){1,63}@successive[.]tech$/gmi;
+let email;
+
+let validEmails=[];
+let invalidEmails=[];
+
+function validateEmail(email)
+{
+    if(email.match(regex)) 
+    {
+                                                                 // console.log("true");
+        return true;
+    }
+    else
+    {
+                                                                 // console.log("false");
+        return false; 
+    }
+
+}
+
+
+function validateUser(user)
+{
+   user.forEach(function(value,index)
+    { 
+        
+        const {traineeEmail,reviewerEmail}=user[index];
+         // console.log(traineeEmail);
+        //console.log(reviewerEmail);
+        
+        if(validateEmail(traineeEmail)){
+            validEmails.push(traineeEmail);
```

Comment:
don't try to access variables which are defined outside of the function, please declare them inside